package.json

```diff
@@ -106,6 +106,7 @@
   "devDependencies": {
     "babel-eslint": "^6.0.0",
     "babel-plugin-add-module-exports": "^0.2.0",
+    "babylon": "^6.15.0",
```

Comment:
As far as I know we can use babel as a parser, without introducing additional dependency. Just run babel without transforms and it will return an AST.